x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/engine_overview.test.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import '../../../__mocks__/shallow_useeffect.mock';
+import { setMockValues, setMockActions } from '../../../__mocks__/kea.mock';
+
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { Loading } from '../../../shared/loading';
+import { EmptyEngineOverview } from './engine_overview_empty';
+import { EngineOverviewMetrics } from './engine_overview_metrics';
+import { EngineOverview } from './';
+
+describe('EngineOverview', () => {
+  const values = {
+    dataLoading: false,
+    documentCount: 0,
+    myRole: {},
+    isMetaEngine: false,
+  };
+  const actions = {
+    pollForOverviewMetrics: jest.fn(),
+  };
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    setMockValues(values);
+    setMockActions(actions);
+  });
+
+  it('renders', () => {
+    const wrapper = shallow(<EngineOverview />);
+    expect(wrapper.find('[data-test-subj="EngineOverview"]')).toHaveLength(1);
+  });
+
+  it('initializes data on mount', () => {
+    shallow(<EngineOverview />);
+    expect(actions.pollForOverviewMetrics).toHaveBeenCalledTimes(1);
+  });
+
+  it('renders a loading component if async data is still loading ', () => {
```

Comment:
```suggestion
  it('renders a loading component if async data is still loading', () => {
```

Reply:
🦅 Thanks for the awesome catch!

Reply:
https://github.com/elastic/kibana/pull/83504/commits/7c6c8c8d3937c6c081603976781e6a5d7d07c2fa